src/protocols/cloud/msger.ts

```
@@ -75,15 +77,149 @@
+
+class RequestAction<Q extends MsgBase, S extends MsgBase> implements Thenable<MsgWithError<S>> {
+  readonly type = "request";
+  readonly req: Q;
+  readonly opts: RequestOpts;
+  readonly realConn: MsgRawConnection;
+  constructor(realConn: MsgRawConnection, req: Q, opts: RequestOpts) {
+    this.req = req;
+    this.opts = opts;
+    this.realConn = realConn;
+  }
+  then(): Promise<MsgWithError<S>> {
```

Biome (1.9.4)

[error] Do not add then to a class.



(lint/suspicious/noThenProperty)